src/preamble.js

```diff
@@ -617,7 +617,7 @@ function abort(what) {
 
 #include "memoryprofiler.js"
 
-#if ASSERTIONS && !('$FS' in addedLibraryItems) && !ASMFS
+#if ASSERTIONS && !('$FS' in addedLibraryItems) && !ASMFS && !WASMFS
```

Comment:
Assuming the plan is to completely remove/replace ASMFS eventually can we remove ASMFS first to avoid having them in tree at the same time?

Reply:
May put this in another PR if needed?

Reply:
Yes, I just wonder if that PR should land first?  i.e. should we cleanup the old code before landing new stuff?

Reply:
We haven’t discussed removing asmjs with @juj yet, so keeping it in for now was deliberate.